server/init.lua

```diff
@@ -23,17 +26,53 @@ event.listen("component_added", updateMachineList)
 
 local statuses = {multiblocks = {}, power = {}}
 
+local function updateMachineStatuses(_evName, _localAddress, sender, port, _distance, ...)
```

Comment:
You can do (_, _, _, port, _, ...) to dismiss the parameters.
Saves a bit of memory (miniscule amounts).